alien4cloud-rest-api/src/main/java/alien4cloud/rest/deployment/DeploymentTopologyDTOBuilder.java

```diff
@@ -137,9 +137,9 @@ private DeploymentTopologyDTO build(FlowExecutionContext executionContext) {
         deploymentTopologyDTO.setLocationPolicies(matchingConfiguration.getLocationIds());
         // Good enough approximation as it doesn't contains just the location dependencies.
         deploymentTopology.setLocationDependencies(executionContext.getTopology().getDependencies());
-        if (executionContext.getExecutionCache().get(FlowExecutionContext.MATCHING_ORIGINAL_NODES) == null) {
-            return deploymentTopologyDTO;
-        }
+        // if (executionContext.getExecutionCache().get(FlowExecutionContext.MATCHING_ORIGINAL_NODES) == null) {
```

Comment:
Do not comment this. Either remove if you are sure that we should not return here or keep it.